src/lib/media-import/status.ts

```diff
@@ -286,8 +280,8 @@ ${ maybeExitPrompt }
 		setProgressTrackerSuffix();
 		progressTracker.print();
 
-		const fileErrors = results.failureDetails?.fileErrors;
-		if ( !! fileErrors && fileErrors.length > 0 ) {
+		const fileErrors = results.failureDetails?.fileErrors ?? [];
+		if ( fileErrors.length > 0 ) {
```

Comment:
The new ESLint rules converted `!! fileErrors` into `Boolean(fileErrors)` and that caused TypeScript to complain.